Product doc typo.mdx (#7248)
getsentry/sentry-docs: src/docs/product/session-replay/replay-details.mdx

## Delete Replays

> Delete an individual replay by clicking the “Delete” button in the top-right corner of **the Replay Details** page. You can’t delete replays that are in-progress. In the event of a PII (Personal Identifiable Information) leak where you need to **bulk** delete replays, contact [support@sentry.io](mailto:support@sentry.io). Please note that deleting replays do not affect your quota.

Reviewer: ```suggestion
Delete an individual replay by clicking the “Delete” button in the top-right corner of **the Replay Details** page. You can’t delete replays that are in-progress. In the event of a PII (Personal Identifiable Information) leak where you need to **bulk** delete replays, contact [support@sentry.io](mailto:support@sentry.io). Please note that deleting replays don't affect your quota.
```
